spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/resources/META-INF/spring-cloud-stream/rabbit-binder.properties

```diff
@@ -18,6 +18,6 @@ spring.cloud.stream.binder.rabbit.default.prefetch:  1
 spring.cloud.stream.binder.rabbit.default.replyHeaderPatterns:       STANDARD_REPLY_HEADERS,*
 spring.cloud.stream.binder.rabbit.default.republishToDLQ:    false
 spring.cloud.stream.binder.rabbit.default.requestHeaderPatterns:     STANDARD_REQUEST_HEADERS,*
-spring.cloud.stream.binder.rabbit.default.requeue:   true
+spring.cloud.stream.binder.rabbit.default.requeueRejected:   true
 spring.cloud.stream.binder.rabbit.default.transacted:false
-spring.cloud.stream.binder.rabbit.default.txSize:    1
+spring.cloud.stream.binder.rabbit.default.txSize:    1
```

Comment:
while we're in here, should rename the default prefix 'xdbus.' -> 'binder.' or 'binder.rabbit.'